app/forms.py

```diff
@@ -31,12 +32,16 @@ def validate_password(self, password):
         if len(password.data) < 6:
             raise ValidationError('Пароль должен содержать минимум 6 символов')
 class UserProfileForm(FlaskForm):
-    surname= StringField('Surname')
-    name= StringField('Name')
-    middlename= StringField('Middlename')
-    address=StringField('Addres')
-    phone=StringField('Phone')
-    info=StringField('Text')
+    surname= StringField('Surname',validators=[DataRequired(message='Заполните поле фамилия!')])
+    name= StringField('Name',validators=[DataRequired(message='Заполните поле имя!')])
+    middlename= StringField('Middlename',validators=[DataRequired(message='Заполните поле отчество!')])
+    address=StringField('Addres',validators=[DataRequired(message='Заполните поле адрес!')])
+    phone=StringField('Phone',validators=[DataRequired(message='Заполните поле телефон!'),Regexp('^[8][\d]{10}', message = "Неправильно введен номер!")])
```

Comment:
Неправильная регулярка, нужно исправить; Так же добавить подсказку о правильном формате: вместо "Неправильно введен номер!" лучше "Введите номер в формате 8XXXXXXXXXX"
```suggestion
    phone=StringField('Phone',validators=[DataRequired(message='Заполните поле телефон!'),Regexp('^[8][\d]{10}$', message = "Неправильно введен номер!")])
```